### DISCUSSION TITLE
[Released] PR for Typo

### DISCUSSION THREAD

**Discussion by mannybecerra** [vercel/community#1406] [Idea]
Hey y'all 👋🏽 

Reaching out here first before submitting a PR (and per the contribution guidelines). 

I came across a super minor typo on the Vercel serverless-functions docs as I was refreshing myself with things on this topic. Cool if I get a PR in while I'm on it or do you prefer your end to handle this? 

Gracias! - M

**Comment by mannybecerra** [comment]
For further context on the typo 👇🏽

![CleanShot 2023-01-31 at 12 51 50@2x](https://user-images.githubusercontent.com/39503964/215880443-f6ff9e6a-da95-4756-946e-e26507b6adbd.png)

**Reply by amyegan** [reply]
Good catch, Manny! I just opened a new PR to fix this. Thank you for reporting it 😄

**Reply by mannybecerra** [reply]
My pleasure Amy, and thanks for the tag team (PR on your end!) 🙌🏼
